server/liveblog/syndication/validation.py

```diff
@@ -14,16 +15,18 @@ def _validate_httpsurl(self, httpsurl, field, value):
 
             key_field = httpsurl.get('key_field')
             check_auth = httpsurl.get('check_auth')
+            chech_auth_enabled = app.config.get('SYNDICATION_VALIDATE_AUTH', False)
```

Comment:
@djbrd-sourcefabric SYNDICATION_VALIDATE_AUTH should be documented as well. This is required to let the testing and development environments work (behave, e2e tests). We will need to setup a dedicated unit testing suite for syndication.